### ISSUE TITLE
Some error can't be parsed in track 2 SDK

### ISSUE THREAD

**Issue by ms-henglu** [Azure/azure-sdk-for-go#19956]
### Bug Report

- import path of package in question: `.../sdk/azcore/arm/runtime/`
- SDK version:`latest`
- output of `go version`

Error like 
```json
{
  "code": "MaxNumberOfRegionalEnvironmentsInSubExceeded",
  "message": "The subscription '00000000-0000-0000-0000-000000000000' cannot have more than 1 environments in East US."
}
```

can not be parsed into https://github.com/Azure/azure-sdk-for-go/blob/688b7e546e1ea83761d939712a9726d7186e411f/sdk/azcore/arm/runtime/policy_register_rp.go#L96

**Comment by ms-henglu** [comment]
related to https://github.com/Azure/terraform-provider-azapi/issues/195

**Comment by jhendrixMSFT** [comment]
Fix will go out in `v1.3.2`
